### ISSUE TITLE
[docs] Splashscreen example missing key argument

### ISSUE THREAD

**Issue by lewisjr** [tauri-apps/tauri#7903]
I wanted to get my splashcreen and main app window as seperate windows running, and do a few custom checks and setups at the Rust level as well as only show the main window once the DOM has loaded. As per the v1 [docs](https://tauri.app/v1/guides/features/splashscreen/#waiting-for-webpage) this is very possible. However, the example does not work out the box. The example on the docs is:

```rust
use tauri::Manager;
// Create the command:
// This command must be async so that it doesn't run on the main thread.
#[tauri::command]
async fn close_splashscreen() {
  // Close splashscreen
  window.get_window("splashscreen").expect("no window labeled 'splashscreen' found").close().unwrap()
  // Show main window
  window.get_window("main").expect("no window labeled 'main' found").show().unwrap();
}
```

This logic almost works but is missing the `window` arg. Without it I got the following errors:

<details>
<summary>**Rust Analyser in VS Code**</summary>

```bash
cannot find value `window` in this scope
not found in this scope
```
</details>  

<details>
<summary>**Terminal**</summary>  

```bash
error[E0425]: cannot find value `window` in this scope
  --> src\utils.rs:10:5
   |
10 |     window
   |     ^^^^^^ not found in this scope

error[E0425]: cannot find value `window` in this scope
  --> src\utils.rs:16:5
   |
16 |     window
   |     ^^^^^^ not found in this scope

warning: unused import: `tauri::Manager`
 --> src\utils.rs:1:5
  |
1 | use tauri::Manager;
  |     ^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0425`.
```
</details>  

To find the solution, I had to do a couple of searches to find [this](https://github.com/tauri-apps/tauri/issues/2185) issue on GitHub, which after poking around led me to [this commit](https://github.com/tauri-apps/tauri/pull/2186/commits/b2a8bc17314bfffbcc037930e1b7e5f0e32a6249). 

Long story short, the solution that worked for me after reading the commit was to add the `window` arg:

```rust
use tauri::Manager;

/// Close the splash screen and open the main window
#[tauri::command]
pub fn init_main(window: tauri::Window) {
    // Close splashscreen
    window.get_window("splash").expect("no window labeled 'splash' found").close().unwrap();
    // Show main app window
    window.get_window("main").expect("no window labeled 'main' found").show().unwrap();
}
```

### System Info
**tauri version:** 1.4
**os:** windows
**windows version:** 10.0.19045
